galley/pkg/fs/fssource_test.go

```diff
@@ -326,16 +326,17 @@ func TestFsSource_DeletePartResorceInFile(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Unexpected error: %v", err)
 	}
-
-	log := logChannelOutput(ch, 4)
-	expected := "[Event](Deleted: [VKey](type.googleapis.com/istio.policy.v1beta1.Rule:some.mixer.rule @v0))"
-
-	if log != expected {
-		t.Fatalf("Event mismatch:\nActual:\n%s\nExpected:\n%s\n", log, expected)
+	//In some OS, it will trigger file deletion event
+	len := len(ch)
+	if len != 8 {
+		log := logChannelOutput(ch, 4)
```

Comment:
why the `len` should equal `8`? In which case?

Reply:
sometimes, it will trigger file deletion event, in this case, it will get 8 events in total

Reply:
I think this is highly unclear, and it is not easy to spot what is going on.
Would it be possible to do a "contains" lookup instead of exact equality?